static/scenes/railroad/index.html

```diff
@@ -31,6 +31,7 @@
 <div id="content">
   <button class="canvas-button" aria-label="throw"></button>
   <img src="img/tutorial-hand.png" class="tutorial-hand">
+  <div class="score-animation"></div>
```

Comment:
Accessibility consideration -- I'm not sure how tools handle this but it might be a little odd for screen readers because they can focus on this new element and then it'll disappear quickly. Something worth testing I guess. We can turn off screen readers reading it with `role = "presentation`